app/assets/javascripts/initializePage.js

```diff
@@ -58,6 +56,8 @@ function initializePage() {
       if (document.getElementById('sidebar-additional')) {
         document.getElementById('sidebar-additional').classList.add('showing');
       }
+      initializePodcastPlayback();
+      initializeVideoPlayback();
```

Comment:
The difference here is that these initializations are now in the block that only executes when the page has loaded correctly (when `data-loaded` is `true`).